tests/sentry/api/endpoints/test_sentry_app_details.py

```diff
@@ -134,17 +133,39 @@ def test_update_unpublished_app(self):
             data={
                 'name': 'NewName',
                 'webhookUrl': 'https://newurl.com',
-                'scopes': ('project:read',)
+                'scopes': ('event:read',),
+                'events': ('issue',),
             },
             format='json',
         )
 
         assert response.status_code == 200
         assert response.data['name'] == 'NewName'
-        assert response.data['scopes'] == ['project:read']
+        assert response.data['scopes'] == ['event:read']
+        assert response.data['events'] == set(['issue'])
```

Comment:
It's weird that this becomes a `Set` in the response. This makes me distrust either `self.client` or django endpoint tests. There's no way JSON would become a `Set`, so something isn't making an _actual_ request in some way.